Documentation/usingupm.md

```diff
@@ -0,0 +1,74 @@
+# Mixed Reality Toolkit and Unity Package Manager
+
+Starting with version 2.5.0, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager (UPM), on Unity 2019.4 and newer. Using UPM, the MRTK gains dependency management and a simple update path, as will be described in the following sections.
+
+## Installing Mixed Reality features using the Unity Package Manager
+
+The Unity Package Mananger uses a manifest file (manifest.json) to determine which packages to install and the registries (servers) from which they can be installed.
+
+> [!Note]
+> As of version 2.5.0 of the MRTK, initial registration of the server and packages is a per-project, manual procedure, please read the following sections for detailed instructions.
+
+### Registering the Mixed Reality component server
+
+For each project that will be using the Microsoft Mixed Realuty Toolkit, the `manifest.json` file (in the Packages folder) will need to have the Mixed Reality scoped registry added. The following illustrate how to properly modify `manfest.json` to support Mixed Reality.
+
+1. Open `<projectRoot>/Packages/manifest.json` in a text editor, such as [Visual Studio Code](https://code.visualstudio.com/).
+1. At the top of the manifest file, add the Mixed Reality server to the scoped registry section and save the file.
+
+```
+{
+  "scopedRegistries": [
+    {
+      "name": "Microsoft Mixed Reality",
+      "url": "https://pkgs.dev.azure.com/aipmr/MixedReality-Unity-Packages/_packaging/Unity-packages/npm/registry/",
+      "scopes": [
+        "com.microsoft.mixedreality",
+        "com.microsoft.spatialaudio"
+      ]
+    }
+  ],
+  "dependencies": {
```

Comment:
Do you think people will copy paste this entire section and inadvertently copy the dependencies line as well?

Reply:
it's possible. i'll remove it

Reply:
done